### ISSUE TITLE
[Question] I finetuning llava1.5-7b model using lora on llava_mix_665k dataset but get  545 on MME perception score. Has anyone else encountered this issue?

### ISSUE THREAD

**Issue by OliverLeeXZ** [haotian-liu/LLaVA#1461]
### Question

I finetuning llava1.5-7b model using lora on llava_mix_665k dataset on 4*A100-40g. However, model has bad performance on MME benchmark. perception score:545, cognition score 197. Has anyone else encountered this issue?

**Comment by OliverLeeXZ** [comment]
@haotian-liu My training hyperparameters remain consistent with you provided. Here are my partial train logs and MME results:
100%|██████████| 10396/10396 [22:39:17<00:00,  9.15s/it]                                             
{'train_runtime': 81561.6745, 'train_samples_per_second': 8.157, 'train_steps_per_second': 0.127, 'train_loss': 3.424769993772549, 'epoch': 1.0}

=========== Perception ===========
total score: 545.515306122449 

	 existence  score: 48.333333333333336
	 count  score: 50.0
	 position  score: 48.333333333333336
	 color  score: 55.00000000000001
	 posters  score: 46.59863945578232
	 celebrity  score: 55.0
	 scene  score: 55.0
	 landmark  score: 68.25
	 artwork  score: 71.5
	 OCR  score: 47.5


=========== Cognition ===========
total score: 197.5 

	 commonsense_reasoning  score: 45.0
	 numerical_calculation  score: 57.5
	 text_translation  score: 75.0
	 code_reasoning  score: 20.0

**Comment by zjysteven** [comment]
@OliverLeeXZ Hi just curious have you found out the reason?

**Comment by HuangChiEn** [comment]
not sure, because of Lora ? (seems finetune will be better)

**Comment by OliverLeeXZ** [comment]
I solving this issue by changing learning rate from 1e-4 to 1e-5. Emm... maybe result by different gpu setting or python packages version?

**Comment by HuangChiEn** [comment]
> I solving this issue by changing learning rate from 1e-4 to 1e-5. Emm... maybe result by different gpu setting or python packages version?

Hi, i also curious that does the lower learning rate, which solve MME drop issue, for finetuning stage only, or for pretrain & finetune stage ?

**Comment by OliverLeeXZ** [comment]
For finetuning stage using LoRA.

**Comment by HuangChiEn** [comment]
> For finetuning stage using LoRA.

got it! TKS for your reply ~

**Comment by junha1125** [comment]
Hi just curious did you set the total batch as `256` for 1 iteration? for example 4 GPU x 16 batch x 4 gradient cumulation.

**Comment by HuangChiEn** [comment]
> Hi just curious did you set the total batch as `256` for 1 iteration? for example 4 GPU x 16 batch x 4 gradient cumulation.

Un...no. I just exactly follows llava v1.5 paper, the appendix have provide batch size (and other hyper-params).
In finetune stage, my global bz is 128. 
(the way you calculate bz is correct BTW, i can not fit batch in GPU, accumulate grad is required~)
![Image](https://github.com/user-attachments/assets/00ae02ea-73ac-4787-8d89-2d315359f8b4)

**Comment by junha1125** [comment]
> > Hi just curious did you set the total batch as `256` for 1 iteration? for example 4 GPU x 16 batch x 4 gradient cumulation.
> 


Thank you for a reply. I thoght 256 bz is needed for finetuning, but 128 is right.
